msgcntr/messageforums-hbm/src/java/org/sakaiproject/component/app/messageforums/dao/hibernate/TopicImpl.java

```diff
@@ -396,4 +399,24 @@ public void setPostFirst(Boolean postFirst) {
 		this.postFirst = postFirst;
 	}
 
+	public Boolean getPostAnonymous() {
+		return isAnonymousEnabled() && postAnonymous;
+	}
+
+	public boolean isAnonymousEnabled() {
+		return ServerConfigurationService.getBoolean("msgcntr.forums.anonymous.enable", true);
```

Comment:
Like my earlier comment, sakai.property and default should probably go in static finals somewhere.